GitCommands/Git/GitModule.cs

```diff
@@ -746,7 +746,7 @@ public async Task<List<ConflictData>> GetConflictsAsync(string filename = "")
                 string hash = findSecondWhitespace >= 0 ? fileStage.Substring(0, findSecondWhitespace).Trim() : "";
                 fileStage = findSecondWhitespace >= 0 ? fileStage.Substring(findSecondWhitespace).Trim() : "";
 
-                if (fileStage.Length > 2 && int.TryParse(fileStage[0].ToString(), out var stage) && stage >= 1 && stage <= 3)
+                if (fileStage.Length > 2 && int.TryParse(fileStage[0].ToString(), out var stage) && stage is >= 1 and <= 3)
```

Comment:
I don't find this syntax particularly better than the old one. I'm finding it requires me to pause and think about it...

Reply:
I learned to like this quite fast. Initially I looked for || and && but this is shorter so it is faster to read and to understand (even better if parens were allowed).